app/components/ExtraSettings.vue

```diff
@@ -1,41 +1,51 @@
 <template>
-<div>
-  <div class="section">
-    <div class="section-content">
-      <p>{{ $t('If you are experiencing weird behavior, you can try deleting your cache directory.  This will result in you losing your scene configuration and settings, but can fix some stability issues.')}}</p>
-      <div class="input-container">
-        <a class="link" @click="showCacheDir">
-          <i class="icon-view" /> <span>{{ $t('Show Cache Directory')}}</span>
-        </a>
-      </div>
-      <div class="input-container">
-        <a class="link" @click="deleteCacheDir">
-          <i class="icon-trash" /> <span>{{ $t('Delete Cache and Restart') }}</span></a>
-      </div>
-      <div class="input-container">
-        <a class="link" @click="uploadCacheDir" :disabled="cacheUploading">
-          <i class="fa fa-upload" /> <span>{{ $t('Upload Cache to Developers') }}</span>
-          <i class="fa fa-spinner fa-spin" v-if="cacheUploading" />
-        </a>
+  <div>
```

Comment:
Diff is unnecessarily large here because you added a level of indentation to everything.  It's actually intentional that the first level of HTML inside the template tag isn't indented.  It's a pattern used by at least a sizable chunk of the Vue community.  I think it's meant to save a level of indentation.  TBH I don't really care much either way, but this would be easier to review without the additional indentation, and many of our files are written this way.

Reply:
that was my linter autoformatting on save...

i guess i can try to configure it to ignore vue templates?

Reply:
AFAIK we don't have any linting configured for `.vue` files.  I'm honestly fine with us ditching this convention, but we have to choose again between making the change all at once (and losing git blamability), or dirtying the diff of every PR that touches a Vue template for a while.